### PR TITLE
Fixed some spelling mistakes

### PR DESCRIPTION
**Pull request by akshaybengani**
Please note that for legal reasons we require you to sign the 
[Contributor Agreement](https://www.arangodb.com/documents/cla.pdf)
before we can accept your pull requests.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -6,18 +6,18 @@ ArangoDB
...
+documents, graphs, and key-values. Build high-performance applications using a
 convenient SQL-like query language or JavaScript extensions. Use ACID
 transactions if you require them. Scale horizontally with a few mouse clicks.
 
 The supported data models can be mixed in queries and allow ArangoDB to be the
 aggregation point for your data.
 
 To get started, try one of our 10 minutes [tutorials](https://www.arangodb.com/tutorials)
-in your favorite programming language or try one of our [ArangoDB Cookbook recipes](https://docs.arangodb.com/cookbook).
+in your favourite programming language or try one of our [ArangoDB Cookbook recipes](https://docs.arangodb.com/cookbook).
```
Comment (Simran-B) [inline README.md]: We use American English (or try to), so this is correct.

File: README.md [hunk 2]
```diff
@@ -6,18 +6,18 @@ ArangoDB
...
 
 For the impatient: [download](https://www.arangodb.com/download) and install
-ArangoDB. Start the server `arangod` and point your browser to `http://127.0.0.1:8529/`.
+ArangoDB. Start the server `arranged` and point your browser to `http://127.0.0.1:8529/`.
```
Comment (Simran-B) [inline README.md]: The server binary is called `arangod`.

File: README.md [hunk 3]
```diff
@@ -58,11 +58,11 @@ Other features of ArangoDB include:
   application-logic and database together for maximal throughput
 - JavaScript for all: **no language zoo**, you can use one language from your
   browser to your back-end
-- **Flexible data modeling**: model your data as combination of key-value pairs,
+- **Flexible data modelling**: model your data as a combination of key-value pairs,
```
Comment (Simran-B) [inline README.md]: American English again.

File: README.md [hunk 4]
```diff
@@ -58,11 +58,11 @@ Other features of ArangoDB include:
...
   documents or graphs - perfect for social relations
 - Different **storage engines**: ArangoDB provides a storage engine for mostly
-  in-memory operations and an alternative storage engine based on RocksDB which 
-  handle datasets that are much bigger than RAM.
+  in-memory operations and an alternative storage engine based on RocksDB which handle
+datasets that are much bigger than RAM.
```
Comment (Simran-B) [inline README.md]: 80 chars width is not a strict rule for us. Also, you missed to indent the wrapped line properly.

File: README.md [hunk 5]
```diff
@@ -72,8 +72,8 @@ Other features of ArangoDB include:
   durability or more performance
 - **Schema-free schemata** let you combine the space efficiency of MySQL with the
   performance power of NoSQL
-- Free **index choice**: use the correct index for your problem, be it a skiplist 
-  or a fulltext search
+- Free **index choice**: use the correct index for your problem, be it a skip list 
+  or a full text search
```
Comment (Simran-B) [inline README.md]: The index types are called "skiplist" and "fulltext" internally and it's actually easier to grep for these terms without false positives if they are written as single word. Not sure if we want to change this. @sleto-it?

**Comment by akshaybengani** [comment]
Added your Prescribed changes

**Comment by dothebart** [comment]
Please don't waste other peoples time just to get some scores in hacktoberfest. Come back again if you have a valuable contribution.

**Pull request closed without merging**